### PR TITLE
Movind all the project details into one root object

### PR DESCRIPTION
**Pull request by DavidGershony**

### CODE REVIEW

**Review by otherdeniz — APPROVED** [review]
looks good

**Comment by otherdeniz** [comment]
... still merge conflicts to solve

File: src/Angor/Client/Models/FlattenedProjectTest.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
+using Angor.Shared.Models;
+
+namespace Angor.Client.Models;
+
+public class FlattenedProjectTest
```
Comment (dangershony) [inline src/Angor/Client/Models/FlattenedProjectTest.cs:5]: What's this a test class?

File: src/Angor/Client/Pages/Invest.razor [hunk 2]
```diff
@@ -220,12 +220,12 @@
...
     bool founder = false;
     bool invested = false;
-    private ProjectInfo? project;
+    private Project? project;
     private bool showCreateModal;
     Transaction? signedTransaction;
     Transaction unSignedTransaction;
 
-    SignatureInfo? recoverySigs;
+    //SignatureInfo? recoverySigs;
```
Comment (dangershony) [inline src/Angor/Client/Pages/Invest.razor:228]: At some point we should clean all this

File: src/Angor/Shared/Models/SignatureInfo.cs [hunk 3]
```diff
@@ -3,15 +3,15 @@ namespace Angor.Shared.Models;
 public class SignatureInfo
 {
     public string ProjectIdentifier { get; set; }
-    public string TransactionId { get; set; }
-    public string RecoveryTransactionId { get; set; }
-    public string RecoveryReleaseTransactionId { get; set; }
-    public string EndOfProjectTransactionId { get; set; }
+    // public string TransactionId { get; set; }
+    // public string RecoveryTransactionId { get; set; }
```
Comment (dangershony) [inline src/Angor/Shared/Models/SignatureInfo.cs:7]: just delete those no?

**Review by dangershony — COMMENTED** [review]
Looks good also from me just some minor changes, where all the pages tested?

File: src/Angor/Client/Models/FlattenedProjectTest.cs (hunk 1 above)
Comment (DavidGershony) [reply src/Angor/Client/Models/FlattenedProjectTest.cs:5]: Testing flattening the root object, not in use for now

**Review by dangershony — APPROVED** [review]

**Pull request merged**